bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/dto/Customer.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.linky.internal.dto;
+
+import org.eclipse.jetty.jaas.spi.UserInfo;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link UserInfo} holds informations about energy delivery point
+ *
+ * @author Gaël L'hopital - Initial contribution
+ * @author Laurent Arnal - Rewrite addon to use official dataconect API
+ */
+
+public class Customer {
+    @SerializedName("customer_id")
```

Comment:
Adding `setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)` could save you from using `@SerializedName` in many places. Also these classes could become `records`.

Reply:
I've made the change.

Reply:
Gael,

I finally remove the setFieldNamingPolicy, because it was conflicting with the deserialization use on the old Web API.